src/@ionic-native/plugins/hyper-track/index.ts

```diff
@@ -0,0 +1,246 @@
+import { Injectable } from '@angular/core';
+import { Cordova, Plugin, IonicNativePlugin } from '@ionic-native/core';
+
+/**
+ * @name HyperTrack
+ *
+ * @description HyperTrack cordova plugin wrapper for Ionic Native. Location-based services provider. iOS is being implemented.
+ *
+ * @see {@link https://docs.hypertrack.com/sdks/cordova/reference.html#methods|Hypertrack Cordova Methods Reference}
```

Comment:
Refrain from using `@see` and `@link` tags. They are not supported here. You can use  Markdown to link to external resources if you need to.